src/screens/WhotoFollow/index.js

```diff
@@ -42,6 +42,7 @@ const VIEW_TYPE_LABEL_LOCATION = 2;
 const VIEW_TYPE_DATA = 3;
 
 const WhotoFollow = () => {
+  // TODO: Change this into useUserAuthHook
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The TODO comment indicates that `useProfileHook` should be replaced with `useUserAuthHook`. However, this change has not been made. To align with the rest of the changes in this PR, consider replacing `useProfileHook` with `useUserAuthHook`.
```diff
-   const {setProfileId} = useProfileHook();
+   const {setProfileId} = useUserAuthHook();
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->